src/gui/Src/Bridge/Bridge.cpp

```diff
@@ -1053,6 +1053,9 @@ void* Bridge::processMessage(GUIMSG type, void* param1, void* param2)
 
     case GUI_GET_MAIN_THREAD_ID:
         return (void*)mMainThreadId;
+    case GUI_UPDATE_GRAPH_VIEW:
+        emit loadCurrentGraph();

```

Comment:
This is already handled in `Bridge::doUpdate` (which gets throttled properly). This can be implemented much simpler by calling `loadCurrentGraphSlot()` (that name change is good) directly inside `DisassemblerGraphView::updateGraphSlot()`.